tests/cases/compiler/newLineFlagWithCR.ts

```diff
@@ -0,0 +1,4 @@
+// @newline: CR
```

Comment:
It was suggested to add a negative test for an invalid command-line arg but the test framework doesn't seem to support it so I'll remove the test.